concepts/05 Widgets/Scheduler/030 Appointments/015 Appointment Types/030 Recurring Appointments.md

```diff
@@ -140,7 +131,7 @@ In the UI, to mark an appointment as recurring, toggle the *"Repeat"* switcher o
 
 ![Scheduler Recurring Appointment Details](/images/UiWidgets/Scheduler_RecurrentAppointment_Details.png)
 
-These fields will be saved as the **recurrenceRule** field of an appointment object. Note that although a recurring appointment is displayed as several appointments on the timetable, only a single appointment object is saved in the data source.
+The specified values will be saved in the appointment's **recurrenceRule** field. Note that although a recurring appointment is displayed as several appointments on the timetable, only a single appointment object is saved in the data source.
```

Comment:
I'd like to change these two sentences to avoid the passive voice (be saved, is displayed, is saved) if possible. This will clarify who/what performs these actions. In this case, is it the "Scheduler"? For example, is the following technically correct?

`The Scheduler control saves the specified values in the appointment's **recurrenceRule** field. Note that although the control displays a recurring appointment as several appointments on the timetable, it only saves a single appointment object to the data source.`